packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -1189,6 +1189,7 @@ export const QUALITY_LEVELS = {
   HIGH: 'HIGH',
   '360p': '360p',
   '480p': '480p',
+  '540p': '540p',
```

Comment:
`QUALITY_LEVELS` are only used for transcoded meetings. I might be missing something obvious, but looking at the code I can't see how can these "XXXp" values work at all, I see the level is stored in `mediaProperties.remoteQualityLevel` and passed on to internal-media-core and there it is used for munging the SDP to limit the maxFs to specific value, but internal-media-core only supports LOW, MEDIUM, HIGH - see [here](https://sqbu-github.cisco.com/WebExSquared/webrtc-media-core/blob/63cd3b78b1b578f58e8e4564fb8609606efea69f/src/MediaConnection/MediaConnection.ts#L645)

Reply:
the "small" 360p is defined and used only in multistream APIs, this QUALITY_LEVELS thing here is only for transcoded, they are separate, I wouldn't change the meaning of LOW for transcoded to avoid any trouble